src/Icons/src/Iconify.php

```diff
@@ -26,12 +26,14 @@
  */
 final class Iconify
 {
+    private const API_BASE_URI = 'https://api.iconify.design';
```

Comment:
Can you use this in the configuration?

Reply:
Do we really need to make this configurable ? :| 

If "yes", i'd then let a null default value.

But using a const value in configuration ? That seems a bit weird. Not that i don't want, this just surprises me :)

Reply:
It's done